x/group/keeper/keeper.go

```diff
@@ -73,16 +73,26 @@ type Keeper struct {
 	voteByVoterIndex    orm.Index
 
 	router *authmiddleware.MsgServiceRouter
+
+	maxMetadataLength uint64
+}
+
+type KeeperParams struct {
```

Comment:
in a pr I made there is a different design for this, it would be good to coordinate on a single design here. 

im a bigger fan of having GovConfig struct in which we add gov module configs, like this `NewKeeper(store,..,..,..,.., govtypes.DefaultConfig)` instead of `NewKeeper(keeperParams{..,..,...,..,...,.})`

Reply:
@blushi do you think we should proceed this way, rather than having a `KeeperParams` struct ?

Reply:
SGTM

Reply:
Note: I'll create a PR to do the same on gov keeper